Thesis.tex

```diff
@@ -58,26 +58,22 @@ \chapter{Introduction}
 
-only applies to expression, not ideas, procedures, method of operation, etc.
-economic rights: allow the rights owner to derive finacial rewards from the use of his work
-moral rights: protect non-economic interests of the author
+Copyright usually only applies to a practical expression and not ideas, procedures or mode of operation. Copyright includes economical and moral rights. The economical rights allow the rights holder to derive financial reward from the use of protected works. The holder of economic rights can authorize or prohibit reproduction, broadcast, public performance, recording, translation and adaptation of protected works. Moral rights protect non-economic interests of the creator like being credited for their work. 
 
-economic rights holder can authorize/prohibit
-reproduction, broadcast, public performance, recording, translation, adaptation
+According to the berne convention copyright is obtained automatically with the act of creation and usually covers a very long time span. There is no registration needed before one can claim and enforce copyright over their own works. Often voluntary registration is available nonetheless to help solve disputes over ownership. Such registrations can help facilitate financial transactions related to a copyright and assignment or transfer of rights to other parties. There exist a number of organizations offering collective management of copyrights, exercising those rights and related transactions on behalf of the rights owner.
 
-according to the berne convention copyright is obtained automatically, without registration
-usually covers a very long timespan
-voluntary registration is often available, can help solve disputes over ownership, facilitate financial transactions, assignment/transfer of rights
+\subsection{Copyright on the internet}
+Many users of the internet have a number of misconceptions about copyright and how it applies to their usage of internet services. The most important misconceptions are outlined in an article by Vorys at al. \cite{Vorys2013}.
 
-collective management of copyright, exercise of copyrights and similar rights by organizations on behalf of rights owner
+Probably the most common misunderstanding is that everything found on the internet can be freely downloaded and used as the basis of own works. However this is nit true in most cases. As mentioned in chapter \ref{sec:CopyDef} copyright is applied automatically to all eligible works and unlicensed reuse is prohibited unless explicitly authorized by the rights owner. The source of the works and whether you payed for i or not -- for instance an iTunes purchase versus a YouTube download -- is not relevant.
```

Comment:
However, this is not true...

Du verweist auf vorys et al - klarstellen, welche Aussage dort gemacht wird? Ich nehme an es betrifft die Aussage "Probably the most..." ?

unlicensed reuse is prohibited unless explicitly authorized by the rights owner: ist diese Authorisierung nicht per se eine Lizenz, womit es nicht mehr unlicensed reuse ist?

Wheter you paid for it or not